api/observability/v1/output_types.go

```diff
@@ -477,6 +477,12 @@ type Elasticsearch struct {
 	//
 	// Static values can only contain alphanumeric characters along with dashes, underscores, dots and forward slashes.
 	//
+	// In case forwarding logs to the Red Hat Managed Elasticsearch instance index must follow pattern `app-.*`, `infra-.*` and "audit-.*`.
```

Comment:
When forwarding logs to the Red Hat Managed Elasticsearch, the index must  match the pattern `^(app|infra|audit)-write$` where the prefix depends upon the `log_type`.  This requires defining a distinct output for each log type or  distinct pipelines with the `openshiftLabels` filter.  See the product documentation for examples.